### PR TITLE
osx/caffeinate, say, uuidgen: add Polish translation

### PR DESCRIPTION
**Pull request by chanibal**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/osx/caffeinate.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# caffeinate
+
+> Nie pozwól aby macOS się uśpił.
+> Więcej informacji: <https://ss64.com/osx/caffeinate.html>.
+
+- Nie usypiaj przez 1 godzinę (3600 sekund):
+
+`caffeinate -u -t {{3600}}`
+
+- Nie usypiaj dopóki komenda nie zostanie zakończona:
```
Comment (EmilySeville7cfg) [inline pages.pl/osx/caffeinate.md:10]: The following code fix can be applied to the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/caffeinate.md):

> Prevent from [s]leeping until a command completes:

Add short option mnemonics when possible.

File: pages.pl/osx/caffeinate.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`caffeinate -s "{{command}}"`
+
+- Nie zasypiaj dopóki nie przerwiesz przez Ctrl-C:
+
+`caffeinate -i`
```
Comment (EmilySeville7cfg) [inline pages.pl/osx/caffeinate.md:16]: The [`caffeinate -m`](https://github.com/tldr-pages/tldr/blob/main/pages/osx/caffeinate.md) example is missing.

File: pages.pl/osx/say.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> Czyta na głos.
+> Więcej informacji: <https://ss64.com/osx/say.html>.
+
+- Powiedz na głos:
+
+`say "{{Lubię jeździć na rowerze.}}"`
+
+- Przeczytaj z pliku:
+
+`say --input-file={{plik.txt}}`
```
Comment (EmilySeville7cfg) [inline pages.pl/osx/say.md:12]: The following code fix could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/say.md) is updated:

```suggestion
`say --input-file={{ścieżka/do/nazwapliku.txt}}`
```

Wrong syntax for path placeholder is used.

File: pages.pl/osx/say.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`say --voice={{głos}} --rate={{słowa_na_minutę}} "{{Przepraszam Dave, ale nie mogę Ci na to pozwolić.}}"`
+
+- Pokaż listę dostępnych głosów:
+
+`say --voice="?"`
+
+- Powiedz coś Angielskim głosem:
+
+`say --voice=Alex "{{Here's to the Crazy Ones.}}"`
```
Comment (EmilySeville7cfg) [inline pages.pl/osx/say.md:24]: The following code fix could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/say.md) is updated:

```suggestion
`say --voice={{Alex}} "{{Here's to the Crazy Ones.}}"`
```

`Alex` can be converted to a placeholder as `--voice` allows several values to choose.

File: pages.pl/osx/say.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Stwórz plik audio z tekstu:
+
+`say --output-file={{plik.aiff}} "{{Litwo, ojczyzno moja!}}"`
```
Comment (EmilySeville7cfg) [inline pages.pl/osx/say.md:28]: The following code fix could be applied after the original [page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/say.md) is updated:

```suggestion
`say --output-file={{ścieżka/do/nazwapliku.aiff}} "{{Litwo, ojczyzno moja!}}"`
```

Wrong syntax for path placeholder is used.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Changes are requested as there are some errors in provided translations.

File: pages.pl/osx/caffeinate.md (hunk 1 above)
Comment (chanibal) [reply pages.pl/osx/caffeinate.md:10]: Not really possible here, adding it causes only confusion. 

For example, I could add `u[s]ypiaj`, but it's not unique to this command.  
Using `zo[s]tanie`(~becomes, ~changes state to) also feels weird.

File: pages.pl/osx/caffeinate.md (hunk 2 above)
Comment (chanibal) [reply pages.pl/osx/caffeinate.md:16]: Example added, thanks for the catch

File: pages.pl/osx/say.md (hunk 4 above)
Comment (chanibal) [reply pages.pl/osx/say.md:24]: Changed to placeholder, also changed wording a bit and mentioned in the previous command (`--voice=?`) that different voices cause different spoken language. I'll add a PR for this change in english page.

File: pages.pl/osx/say.md (hunk 5 above)
Comment (chanibal) [reply pages.pl/osx/say.md:28]: Fixed

File: pages.pl/osx/say.md (hunk 3 above)
Comment (chanibal) [reply pages.pl/osx/say.md:12]: Fixed

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
@kbdharun, @CleanMachine1, should we merge this PR right now? There is no requirement in the current version of maintainers guide to wait until all conversations are resolved (but this issue should me addressed in #8398 PR).

**Review by kbdharun — COMMENTED** [review]
> Should we merge this PR right now? 

@EmilySeville7cfg I would suggest waiting for some time for the contributor to apply the suggestions as this page isn't in sync with the main page. 

> There is no requirement in the current version of the maintainer's guide to wait until all conversations are resolved (but this issue should be addressed in #8398 PR).

Yes, there isn't any requirement to wait for conversations to be resolved for PRs, But I think we should make a distinction here to differentiate normal page additions PRs and translation PRs. If a translation isn't in sync with the main page, we should wait for the conversation to be resolved IMO instead of merging it and adding it to the pile of translations requiring updations to sync with the main page. I don't think this requires being added to the maintainer's guide, I have no objections anyway.

Edit: Seems like the main page was indeed updated, I haven't refreshed my local cache in a couple of days, So sorry for the confusion.

File: pages.pl/osx/say.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+`say --voice={{głos}} --rate={{słowa_na_minutę}} "{{Przepraszam Dave, ale nie mogę Ci na to pozwolić.}}"`
+
+- Pokaż listę dostępnych głosów, różne głosy obsługują różne języki:
+
+`say --voice="?"`
+
+- Powiedz coś po angielsku:
+
+`say --voice={{Alex}} "{{Here's to the Crazy Ones.}}"`
+
```
[the remark this answers is not in the document]
Comment (chanibal) [reply pages.pl/osx/say.md:25]: (for both comments, sorry misclicked and can't unresolve on mobile)

Unfortunately it's untranslatable. Checked Steve Jobs quotes in polish, but it was always left in English. Any manual translation seemed off (or like a alcohol related toast). So I used a famous polish quote and moved that one to the say in different language section.

About the new section, there is a PR to add it to English: https://github.com/tldr-pages/tldr/pull/9452 There, I used the quotes in reverse.  
And yes, I think it lacked this information. How should I know (without listing voices at least) that to say something in English I have to set not the language code, but call some Alex?

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.pl/osx/say.md:25]: @chanibal Thanks for the clarification, Sorry for the inconvenience I haven't updated the local cache of my tldr client for a couple of days. Seems like you updated it recently so I think this PR is GTG.

**Pull request merged**